configure.ac

```diff
@@ -262,6 +261,32 @@ LD="$saved_LD"
 # AUTOHARDEN END
 #####################################################
 
+#####################################################
+# OPTIMIZATION LEVEL START
+
+saved_CC="$CC"
+saved_LD="$LD"
+flag_wrap="$srcdir/wrap-compiler-for-flag-check"
+CC="$flag_wrap $CC"
+LD="$flag_wrap $LD"
+
+OPTIMIZATION_CFLAGS=""
+
+AS_IF([test x"$hardening" != x"yes"], [
```

Comment:
Why is this incompatibility with hardening?

Reply:
Because with hardening enabled `-O2` is specified

Reply:
Perhaps it would make sense to remove 02 from the harden flags.